### PR TITLE
feat: update semantic release with node

### PR DESCRIPTION
**Pull request by rudouglas**

### CODE REVIEW

**Review by aswanson-nr — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -170,7 +170,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 10.x
+          node-version: 14
 
       - name: Download Cached Deps
```
Comment (aswanson-nr) [inline .github/workflows/release.yml:175]: Let's remove the cached dependencies step, so we don't pollute the cache for the other jobs.

**Review by aswanson-nr — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (aswanson-nr) [inline .github/workflows/release.yml:175]: Okay, it won't let me add a change suggestion to the unedited part of the file. But we should remove this cache step

**Review by aswanson-nr — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (rudouglas) [reply .github/workflows/release.yml]: @aswanson-nr riiiight i should have asked what you meant by that 🏚️ Before I change this on all the rest is my update correct?

**Review by aswanson-nr — APPROVED** [review]
You got it!

**Pull request merged**